packages/block-library/src/query-pagination/edit.js

```diff
@@ -23,5 +51,21 @@ export default function QueryPaginationEdit() {
 		],
 		orientation: 'horizontal',
 	} );
-	return <div { ...innerBlocksProps } />;
+	return (
+		<>
+			{ hasNextPreviousBlocks && (
+				<InspectorControls>
+					<PanelBody title={ __( 'Settings' ) }>
+						<QueryPaginationArrowControls
```

Comment:
If we want to show this control on each of the next/previous buttons, it can be prove to be a good use case for the work @gziolo is doing to show parent inspector controls in nested blocks.

Reply:
Yes, there's a family of overlapping features in expressing attributes in a parent (so it's consistent) while cascading to children in terms of being able to edit in either place.

Reply:
👍 ! That would be good and we would have to show this control to specific nested blocks only (QueryPaginationNext, QueryPaginationPrevious). QueryPagination can have (for now) QueryPaginationNumbers as well, that has nothing to do with the `arrow`.